nimare/meta/kernel.py

```diff
@@ -202,13 +206,11 @@ def transform(self, dataset, masker=None, return_type="image"):
         if return_type == "sparse":
             return transformed_maps[0]
 
+        # Optimize this, possily make own function
         imgs = []
         # Loop over exp ids since sparse._coo.core.COO is not iterable
         for i_exp, id_ in enumerate(transformed_maps[1]):
-            if isinstance(transformed_maps[0][i_exp], sparse._coo.core.COO):
```

Comment:
Apparently this check is necessary, because without it tests fails. However, it _shouldn't_ be necessary because the output should be sparse here. 

It's not in `ALEKernel` however.

This is inconsistent w/ the API because if you say "return_type='sparse'" you could get back a dense matrix w/ an ALEKernel

Reply:
Moved to #692.

Reply:
Sorry that I missed this comment. I created a new folder for GitHub email and I didn't get notified of this one. I added a comment to the new issue.